CHANGELOG.md

```diff
@@ -54,6 +54,7 @@
 * [ENHANCEMENT] Compactor: Introduce cleaner visit marker. #6113
 * [ENHANCEMENT] Query Frontend: Add cortex_query_samples_total metric. #6142
 * [ENHANCEMENT] Ingester: Implement metadata API limit. #6128
+* [ENHANCEMENT] Ruler: Add query statistics metrics when --ruler.query-stats-enabled=true. #6173
```

Comment:
We need to fix changelog. Can we move this to master section?

Reply:
I have moved it.